Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -0,0 +1,176 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Models.CCT.Context;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Gordon360.Authorization;
+
+namespace Gordon360.Services.RecIM
+{
+    public class TeamService : ITeamService
+    {
+        private readonly CCTContext _context;
+        private readonly IMatchService _matchService;
+
+        public TeamService(CCTContext context, IMatchService matchService)
+        {
+            _context = context;
+            _matchService = matchService;
+        }
+
+        public TeamViewModel GetTeamByID(int teamID)
+        {
+
+            var team = _context.Team
+                .Where(t => t.ID == teamID)
+                .Select(t => new TeamViewModel
+                {
+                    ID = teamID,
+                    Name = t.Name,
+                    Status = _context.TeamStatus
+                            .FirstOrDefault(s => s.ID == t.StatusID)
+                            .Description,
+                    Private = t.Private,
+                    Logo = t.Logo,
+                    Match = t.MatchTeam
+                        .Select(mt => _matchService.GetMatchByID(mt.MatchID)),
+                    Participant = t.ParticipantTeam
+                        .Select(pt => new ParticipantViewModel
+                        {
+                            Username = _context.ACCOUNT
+                                .FirstOrDefault(a => Int32.Parse(a.gordon_id) == pt.ParticipantID)
+                                .AD_Username,
+                            Email = _context.ACCOUNT
+                                .FirstOrDefault(a => Int32.Parse(a.gordon_id) == pt.ParticipantID)
+                                 .email,
```

Comment:
Are you looking for `int.Parse(...)`?
```suggestion
                                .FirstOrDefault(a => Int32.Parse(a.gordon_id) == pt.ParticipantID)
                                .AD_Username,
                            Email = _context.ACCOUNT
                                .FirstOrDefault(a => int.Parse(a.gordon_id) == pt.ParticipantID)
                                 .email,
```

Reply:
"I just tested and turned out int.Parse(...) did not work. So instead I used toString() to convert pt.ParticipantID to match the string type rather than converting string to int." - @Kevin-Chu58

Reply:
Yes! I was having the same issue with `int.Parse()` and did use the same method you just mentioned.